scripts/metricsLib/metrics_definitions.py

```diff
@@ -116,6 +119,16 @@
                                  {"total_project_blank_lines": ["blank_lines"],
                                  "average_blank_lines": ["avg_blank_lines"]}))
 
+SIMPLE_METRICS.append(ListMetric("repositoryLanguages", 
+                                 ["owner", "repo"], 
+                                 LANGUAGE_ENDPOINT, 
+                                 {"languages": None}, 
```

Comment:
The dictionary here is supposed to act as a map for the JSON data that is returned, meaning that the key/value pair corresponds to how to save the data that is returned. 

For example, if there was an api that returned the JSON:
 `{ 'apples': 7}`
 You would use a dictionary such as this to represent how to map the data: 
 `{'numApples':'apples'}`. This would make it so that the resulting API data would be returned as:
 `{'numApples':7}` and added to the other data.
 
 This way the smaller data is given a more descriptive label in the larger dictionary returned from the API.
 
 I'm not sure why there is a `None` here.